src/promptflow/promptflow/executor/_dag_manager.py

```diff
@@ -12,6 +13,7 @@ def __init__(self, nodes: List[Node], flow_inputs: dict):
         self._pending_nodes = {node.name: node for node in nodes}
         self._completed_nodes_outputs = {}  # node name -> output
         self._bypassed_nodes = {}  # node name -> node
+        self._bypassed_by_skip_config = []
```

Comment:
Consider that `skip config` will be removed in the future, it would be better not to introduce any new variables related to skip config.